src/components/VideoCard/VideoCard.styles.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+
+const Container = styled.div`
+  border: 10px solid #cccccc;
+  border-radius: 15px;
+  padding: 20px;
+  width: 300px;
+  margin-bottom: 20px;
+  margin: 10px auto;
+`;
+
+const Title = styled.h1``;
+
+const Body = styled.p``;
+
+const Preview = styled.img`
+  width: 100%;
+`;
+
+const Styled = { Container, Title, Body, Preview };
+export default Styled;
```

Comment:
Same comment as before, everything is fine, an improvement you can make is to change the way you export things, it can change to:

```jsx
export const Container = styled.div`
  ...
`;

export const Title = styled.h1``;

export const Body = styled.p``;

export const Preview = styled.img`
  ...
`;
```
and remove the lines 20 & 21, then you can import the components like:

```jsx
import {Container, Title, Body, Preview} from './VideoCard.styles';
```